vnext/Shared/IWebSocketResource.h

```diff
@@ -83,7 +83,7 @@ struct IWebSocketResource {
   /// WebSocket URL address the instance will connect to.
   /// The address's scheme can be either ws:// or wss://.
   /// </param>
-  static std::shared_ptr<IWebSocketResource> Make(std::string &&url);
+  static std::shared_ptr<IWebSocketResource> __cdecl Make(std::string &&url);
```

Comment:
>__cdecl [](start = 45, length = 7)

(nit) it would be nice to have a macro for such specifications, but good here for this checkin

Reply:
Not sure I fully agree.

- Defining a new macro would be, in my opinion, redundant.
- Using a predefined macro from the Windows headers would add an unnecessary header dependency to this rather simple interface.

We can can revisit this topic, also discussing whether the struct's methods should also have an explicit calling convention.